hail/python/hailtop/batch/batch.py

```diff
@@ -343,6 +353,7 @@ def hello(name):
             j.timeout(self._default_timeout)
 
         self._jobs.append(j)
+        self._unsubmitted_jobs.append(j)
```

Comment:
I don't think this is meant to be here anymore since `_unsubmitted_jobs` is a computed property